packages/user-operation-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.0.1]
+
+### Changed
+
+- Bump `@metamask/polling-controller` from `^12.0.0` to `^12.0.1` ([#4870](https://github.com/MetaMask/core/pull/4870))
+- Bump `@metamask/keyring-controller` from `^17.3.0` to `^17.3.1` ([#4870](https://github.com/MetaMask/core/pull/4870))
```

Comment:
This is a peer dependency bump. Can we update this changelog entry and then also bump the version to 18.0.0?

``` suggestion
## [18.0.0]

### Changed

- **BREAKING:** Bump `@metamask/keyring-controller` from `^17.3.0` to `^18.0.0` ([#4915](https://github.com/MetaMask/core/pull/4915))
- Bump `@metamask/polling-controller` from `^12.0.0` to `^12.0.1` ([#4870](https://github.com/MetaMask/core/pull/4870))
```